mergin/client_push.py

```diff
@@ -124,7 +124,7 @@ def push_project_async(mc, directory):
     changes = filter_changes(mc, project_info, changes)
     mp.log.debug("push changes:\n" + pprint.pformat(changes))
 
-    tmp_dir = tempfile.TemporaryDirectory(prefix="python-api-client-")
+    tmp_dir = tempfile.TemporaryDirectory(prefix="python-api-client-", ignore_cleanup_errors=True, delete=True)
```

Comment:
is this a followup for the previous PR to make sure the push temp dirs get cleaned up properly? (or is it related to download or project somehow?)

Reply:
this was missing from previous PR